src/Uno.UI.Runtime.Skia.X11/X11SoftwareRenderer.cs

```diff
@@ -91,11 +91,16 @@ void IX11Renderer.InvalidateRender()
 				bitmap_pad: BitmapPad,
 				bytes_per_line: 0); // 0 bytes per line assume contiguous lines i.e. pad * width
 
+			var image = _xImage.Value;
+
+			_gc ??= X11Helper.XCreateGC(x11window.Display, x11window.Window, 0, 0);
+			var gc = _gc.Value;
+
 			var _4 = X11Helper.XPutImage(
 				display: x11window.Display,
 				drawable: x11window.Window,
-				gc: X11Helper.XDefaultGC(x11window.Display, XLib.XDefaultScreen(x11window.Display)),
```

Comment:
Why not use `X11Helper.XDefaultGC`? Because we may not use the default argument to create the x11 window. The GC from XDefaultGC may break with the error message:

```
X Error of failed request:  BadMatch (invalid parameter attributes)
  Major opcode of failed request:  72 (X_PutImage)
  Serial number of failed request:  91
  Current serial number in output stream:  122
```